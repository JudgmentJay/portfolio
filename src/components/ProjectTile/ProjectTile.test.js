import React from 'react'
import renderer from 'react-test-renderer'

import { ProjectTile } from '../'

const mockData = {
	title: 'Flickr Image Gallery',
	url: 'https://www.jayposten.com/gallery',
	imgName: 'project_gallery',
	testId: 'project-0'
}

it('Renders correctly', () => {
	const tree = renderer
		.create(<ProjectTile {...mockData} />)
		.toJSON()

	expect(tree).toMatchSnapshot()
})
