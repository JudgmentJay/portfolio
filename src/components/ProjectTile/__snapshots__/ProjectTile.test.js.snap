// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
<div
  className="tile"
  data-testid="project-0"
>
  <picture>
    <source
      media="(max-width: 519px)"
      srcSet="img/project_gallery_m.webp"
      type="image/webp"
    />
    <source
      media="(max-width: 519px)"
      srcSet="img/project_gallery_m.jpg"
      type="image/jpeg"
    />
    <source
      media="(max-width: 739px)"
      srcSet="img/project_gallery_t.webp"
      type="image/webp"
    />
    <source
      media="(max-width: 739px)"
      srcSet="img/project_gallery_t.jpg"
      type="image/jpeg"
    />
    <source
      srcSet="img/project_gallery.webp"
      type="image/webp"
    />
    <source
      srcSet="img/project_gallery.jpg"
      type="image/jpeg"
    />
    <img
      alt="Flickr Image Gallery"
      className="image"
      loading="lazy"
      sizes="(max-width: 519px) 460px, (max-width: 739px) 210px, 348px"
      src="img/project_gallery.jpg"
      srcSet="img/project_gallery.jpg 460w, img/project_gallery.jpg 210w, 348w"
    />
  </picture>
  <h4
    className="title"
  >
    Flickr Image Gallery
  </h4>
  <a
    aria-label="Flickr Image Gallery"
    className="link"
    href="https://www.jayposten.com/gallery"
    rel="noreferrer noopener nofollow"
    target="_blank"
  />
</div>
`;
